ui/src/views/options/sections/Funscripts.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <div class="content">
+      
+      <h3>{{$t('Export funscripts')}}</h3>
+      <p>
+        {{$t('Here you can download a zip file with a funscript for each scripted scene. The files are named with the scene title and the scene id. Scipted scenes in the DeoVR interface will be named accordingly. For scenes with multiple scripts you can choose a preferred one in the scene details view. Otherwise the last one added is chosen.')}}
+      </p>
+      <p>
+        {{$t('This export won\'t work with DLNA.')}}
+      </p>
+      <p>
+        {{$t('To use this export with DeoVR: Unzip and put the files in the Interactive folder on your device.')}}
+      </p>
+      <p>
+        {{$t('To use this export with ScriptPlayer: Unzip and put the files in a folder of your choice. Add this folder in the Paths section in the settings. Connect to DeoVR.')}}
```

Comment:
```suggestion
        {{$t('Here you can download a ZIP file containing a funscript for each scripted scene. The files are named by scene title and scene id, just as they appear in the DeoVR interface. For scenes with multiple scripts, you can choose a preferred script in the scene details view. Otherwise, the last added script is chosen.')}}
      </p>
      <p>
        {{$t('Note that this export does not work with DLNA.')}}
      </p>
      <p>
        {{$t('To use this export with DeoVR: Unzip and put the files in the Interactive folder on your device.')}}
      </p>
      <p>
        {{$t('To use this export with ScriptPlayer: Unzip and put the files in a folder of your choice. In the ScriptPlayer settings, add this folder in the Paths section, then connect to DeoVR.')}}
```